src/main/java/domain/LottoMoney.java

```diff
@@ -12,6 +12,14 @@ public LottoMoney(int money) {
 		this.lottoMoney = money;
 	}
 
+	public LottoMoney calculateRemainAfterBuy(int count) {
+		int remain = this.lottoMoney - count * LOTTO_PRICE_UNIT;
+		if (remain < 0) {
+			throw new IllegalStateException("금액이 부족합니다!");
+		}
+		return new LottoMoney(lottoMoney - count * LOTTO_PRICE_UNIT);
```

Comment:
이미 계산해둔 `remain`으로 새로운 LottoMoney 객체를 생성하면될 것 같네요. 🙂
```suggestion
		return new LottoMoney(remain);
```